File: app/src/main/java/com/softhink/single/dashboard/PerfilFragment.java
package com.softhink.single.dashboard;


import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;
import android.widget.TextView;

import com.softhink.single.MapsActivity;
import com.softhink.single.R;
import com.softhink.single.survey.SurveyActivity;

/**
 * A simple {@link Fragment} subclass.
 */
public class PerfilFragment extends Fragment implements View.OnClickListener {

    private TextView textView;
    private View view;
    private Button singlear;


    public PerfilFragment() {
        // Required empty public constructor
    }


    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        view =  inflater.inflate(R.layout.fragment_perfil, container, false);
        return view;
    }

    @Override
    public void onViewCreated(@NonNull View v, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(v, savedInstanceState);
        if (savedInstanceState == null) {
            textView = view.findViewById(R.id.button5);
            singlear = view.findViewById(R.id.singlear);
            textView.setOnClickListener(this);
            singlear.setOnClickListener(this);
        }
    }

    @Override
    public void onClick(View v) {
        Intent intent = null;
        switch (v.getId()){
            case R.id.button5:
                intent = new Intent(getActivity(), SurveyActivity.class);
                break;

            case R.id.singlear:
                intent = new Intent(getActivity(), MapsActivity.class);
                break;
        }
        startActivity(intent);
    }
}

File: app/src/main/java/com/softhink/single/dashboard/PerfilFragment.kt
package com.softhink.single.dashboard

import android.content.Intent
import android.os.Bundle
import androidx.annotation.NonNull
import androidx.annotation.Nullable
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.Button
import android.widget.TextView
import com.softhink.single.R
import com.softhink.single.survey.SurveyActivity


/**
 * A simple [Fragment] subclass.
 */
class PerfilFragment : Fragment(), View.OnClickListener {

    private var editParam: TextView? = null
    private var editProfile: TextView? = null
    private var view: View? = null
    private var singlear: Button? = null


    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        // Inflate the layout for this fragment
        view = inflater!!.inflate(R.layout.fragment_perfil, container, false)
        return view
    }

    override fun onViewCreated(@NonNull v: View?, @Nullable savedInstanceState: Bundle?) {
        super.onViewCreated(v, savedInstanceState)
        if (savedInstanceState == null) {
            editParam = view!!.findViewById(R.id.toEditParams)
            editProfile = view!!.findViewById(R.id.toEditProfile)
            singlear = view!!.findViewById(R.id.singlear)

            editProfile!!.setOnClickListener(this)
            editParam!!.setOnClickListener(this)
            singlear!!.setOnClickListener(this)
        }
    }

    override fun onClick(v: View) {
        var intent: Intent? = null
        when (v.id) {
            R.id.toEditParams -> intent = Intent(activity, SurveyActivity::class.java)

            R.id.toEditProfile -> intent = Intent(activity, EditProfileActivity::class.java)

            R.id.singlear -> intent = Intent(activity, MapsActivity::class.java)
        }
        startActivity(intent)
    }
}// Required empty public constructor
